parcelable/src/commonMain/kotlin/com/arkivanov/essenty/parcelable/TypeParceler.kt

```diff
@@ -0,0 +1,12 @@
+package com.arkivanov.essenty.parcelable
+
+/**
+ * A multiplatform alias for `kotlinx.parcelize.TypeParceler` from the `kotlin-parcelize` Gradle plugin.
+ * Make sure you have the `kotlin-parcelize` Gradle plugin applied.
+ */
+@OptIn(ExperimentalMultiplatform::class)
+@OptionalExpectation
+@Retention(AnnotationRetention.SOURCE)
+@Repeatable
+@Target(AnnotationTarget.CLASS, AnnotationTarget.PROPERTY)
+expect annotation class TypeParceler<T, P : Parceler<in T>>()
```

Comment:
There are some [docs](https://github.com/arkivanov/Essenty#custom-parcelers) in the readme for custom parcelers. Could you please somehow mention `TypeParceler` there as well?